business-central-parent/business-central-webapp/src/main/webapp/WEB-INF/web.xml

```diff
@@ -433,6 +433,21 @@ See http://www.w3.org/TR/SVG/intro.html#MIMEType. -->
     </web-resource-collection>
   </security-constraint>
 
+  <security-constraint>
+    <web-resource-collection>
+      <web-resource-name>rest controller</web-resource-name>
+      <url-pattern>/rest/controller/*</url-pattern>
+      <http-method>GET</http-method>
+      <http-method>PUT</http-method>
+      <http-method>POST</http-method>
+      <http-method>DELETE</http-method>
+    </web-resource-collection>
+    <auth-constraint>
+      <role-name>admin</role-name>
+      <role-name>rest-all</role-name>
```

Comment:
Just to double-check: we don't need the `kie-server` role here too, right?

Reply:
where is kie-server role? It just BC's role in here.

Reply:
Yes, this role is not here. This is exactly what I want to double-check.